### PR TITLE
Change order of miners for rest of CH3

### LINKED ISSUE #623: Reorder cooperators in Ch. 3
In chapter-3/coop-2, we swapped coin crushers and hash hoppers so that hash hoppers would be the ones winning blocks. 

Screenshot 2023-09-03 at 1 45 51 PM

We should keep the same order for chapter-3/pool-3 and chapter-3/split-2
Screenshot 2023-09-03 at 1 46 23 PM

Screenshot 2023-09-03 at 1 46 59 PM

### PR DESCRIPTION
**Pull request by benalleng**
Closes #623

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
I can't seem to test this. I'm running into issues getting past intro-1 again. 

https://saving-satoshi-fzyjn9i7g-savingsatoshi.vercel.app/en/chapters/chapter-3/intro-1?dev=true

**Comment by benalleng** [comment]
dangit! hmm ok, let me see if i need to rebase or something

File: content/lessons/chapter-3/coop-1.tsx [hunk 1]
```diff
@@ -32,13 +32,13 @@ export default function Coop1({ lang }) {
       display: false,
     },
     {
-      username: 'Hash Hoppers',
```
Comment (benalleng) [inline content/lessons/chapter-3/coop-1.tsx:35]: A review

**Review by ecurrencyhodler — APPROVED** [review]
tACK! Tested and it is the correct order across all challenges. I also didn't run into any bugs.

**Pull request merged**